.flowconfig

```diff
@@ -3,6 +3,9 @@
 <PROJECT_ROOT>/packages/react-native/template/.*
 <PROJECT_ROOT>/packages/react-native/sdks/.*
 
+; Ignore the codegen e2e tests
+<PROJECT_ROOT>/packages/react-native-codegen/e2e/__test_fixtures__/modules/NativeEnumTurboModule.js
```

Comment:
flow check failed https://app.circleci.com/pipelines/github/facebook/react-native/46812/workflows/f35b38a2-172b-4f7c-b041-57b63ca9692e/jobs/1524067. Seems `// $FlowFixMe` not work for SyntaxError.